cpp/iedriver/IECommandExecutor.cpp

```diff
@@ -438,6 +439,8 @@ unsigned int WINAPI IECommandExecutor::ThreadProc(LPVOID lpParameter) {
         LOG(DEBUG) << "Returned from DestroyWindow()";
         break;
       } else {
+        static std::mutex messageLock;
```

Comment:
Can we have a comment explaining why the mutex is necessary?

Reply:
Sure!

Reply:
I'll keep on eye on this PR for that :) Thank you!